src/variant.cpp

```diff
@@ -344,6 +344,19 @@ namespace {
         v->checking = false;
         return v;
     }
+    // Dodo Chess
+    // https://www.evochess.com/chess-variant/dodo
+    Variant* dodochess_variant() {
+        Variant* v = chess_variant_base()->init();
```

Comment:
Since dodochess is basically racing kings with a different starting position, I would suggest to inherit from racing kings here. Then only the differences would need to be explicitly mentioned, i.e., the startFen, as well as optionally removing pawns and queens from the set of piece types.